app/controllers/document_lead_image_controller.rb

```diff
@@ -117,6 +117,30 @@ def choose_image
     redirect_to document_path(document)
   end
 
+  def delete_image
+    document = Document.find_by_param(params[:document_id])
+    image = document.images.find(params[:image_id])
+
+    begin
+      if image.id == document.lead_image_id
+        DocumentUpdateService.update!(
+          document: document,
+          user: current_user,
+          type: "lead_image_removed",
```

Comment:
Is a timeline entry type? It doesn't look to be in https://github.com/alphagov/content-publisher/blob/fb0d834d356a980f0e8ff12fb0a4e68895d01d58/app/models/timeline_entry.rb#L7 but I can see it was already used in this controller before so not sure how/if it works properly

Reply:
You're right - it should be `validates_inclusion_of` - will fix in a separate PR.